include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.h

```diff
@@ -210,6 +210,12 @@ class MultigridPreconditionerBase : public PreconditionerBase<Number>
   unsigned int
   get_number_of_levels() const;
 
+  /**
+   * Returns the number of h-levels.
+   */
+  unsigned int
+  get_number_of_h_levels() const;
+
```

Comment:
This function seems to be protected. Did you check whether derived classes make use of this functionality (and then could also call this member function) or - otherwise - whether this function can be made private?

Reply:
made private! To be clear: I introduced this function with this PR. We use it twice currently. Used it four times in previous versions of this PR.